### ISSUE TITLE
aws_lb resource enable_http2 for network type load balancer doesn't make sense (and is invalid)

### ISSUE THREAD

**Issue by tpickett66** [terraform-providers/terraform-provider-aws#4011]
After updating from `1.10` to `1.13` an unexpected change to my NLB is trying to be made.

### Terraform Version
```
» terraform -v
Terraform v0.11.5
+ provider.aws v1.13.0
```

### Affected Resource(s)
- `aws_lb`

### Terraform Configuration Files
```hcl
resource "aws_lb" "my-lb" {
  name_prefix        = "my-lb"
  load_balancer_type = "network"
}
```

### Expected Behavior
I expected the network type LB to not default to enabling http2.

### Actual Behavior
The new default attempted to enable http2 for a network type load balancer.

### Steps to Reproduce
1. Provision a load balancer of type "network" using a v1.10 of the provider
2. Update to 1.13 of the provider
3. Run `terraform plan` and note the addition of the new default.

**Comment by bflad** [comment]
The attributes have had a `DiffSuppressFunc` added via #4032, which was merged into master and will release with v1.14.0 of the AWS provider, likely by the end of the week.

**Comment by bflad** [comment]
This has been released in [version 1.14.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1140-april-06-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
